docs/integrations/saas-cloud/confluent-cloud.md

```diff
@@ -0,0 +1,119 @@
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/saas-cloud/confluent-cloud.png')} alt="Thumbnail icon" width="100"/>
+
+Confluent Cloud is a fully managed, cloud-native data streaming platform based on Apache Kafka. It enables real-time data processing, integration, and movement across applications and systems without managing infrastructure. With built-in support for connectors, ksqlDB, and stream governance, it simplifies building scalable, event-driven architectures.
+
+The Sumo Logic App for Confluent Cloud provides deep visibility into your Confluent Cloud environment based on key platform metrics. The app’s dashboards use preconfigured searches and filters to help you monitor Kafka cluster performance, topic-level throughput, consumer lag, producer latency, connector health, Flink compute pool utilization, KSQL activity, and schema registry operations—enabling proactive troubleshooting and streamlined data streaming operations.
+
+## Metrics type  
+
+The Confluent Cloud app uses metrics:
+* [Confluent Cloud runtime metrics](https://api.telemetry.confluent.cloud/docs/descriptors/datasets/cloud).
+
+### Collect Metrics
+
+Sumo Logic supports collecting metrics by C2C source:
+
+* Configure a [Confluent Cloud Metrics Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/confluent-cloud-metrics-source/)
+
+## Installing the Confluent Cloud  
+
+Now that you have set up a collection for **Confluent Cloud**, install the Sumo Logic app to use the pre-configured [dashboards](#viewing-the-confluent-cloud-dashboards) that provide visibility into your environment for real-time analysis of overall usage.
+
+import AppInstall from '../../reuse/apps/app-install-v2.md';
+
+<AppInstall/>
+
+## Viewing the Confluent Cloud dashboards  
+
+### Confluent Cloud - Compute Pool
+
+Confluent Cloud - Compute Pool dashboard provides details on the status and utilization of compute resources in your Confluent Cloud environment. Use this dashboard to:
+
+* Monitor the average Flink statement status to identify potential issues with stream processing jobs.
+* Track CPU utilization against the set limit to ensure optimal resource allocation and prevent performance bottlenecks.
+* Analyze trends in CPU minutes consumed over time to optimize cost and capacity planning for your Confluent Cloud deployment.
+* Compare current CPU usage across different compute pools to balance workloads and improve overall system efficiency.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ConfluentCloud/Confluent-Cloud-Compute-Pool.png')} alt="Confluent Cloud - Compute Pool" style={{border: '1px solid gray'}} width="800" />
+
+### Confluent Cloud - Connectors
+
+Confluent Cloud - Connectors dashboard provides details on the status, performance, and health of Confluent Cloud connectors.
```

Comment:
apply the same change as https://github.com/SumoLogic/sumologic-documentation/pull/5495/files#r2172504296